src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlServerFhirDataStore.cs

```diff
@@ -407,7 +407,12 @@ public async Task<UpsertOutcome> UpsertAsync(ResourceWrapperOperation resource,
 
         public async Task<IReadOnlyList<ResourceWrapper>> GetAsync(IReadOnlyList<ResourceKey> keys, CancellationToken cancellationToken)
         {
-            return await _sqlStoreClient.GetAsync(keys, _model.GetResourceTypeId, _compressedRawResourceConverter.ReadCompressedRawResource, _model.GetResourceTypeName, cancellationToken);
+            return await GetAsync(keys, false, cancellationToken); // do not retrun invisible records in public interface
```

Comment:
NIT: typo "// _do not **return** invisible records in public interface_".

Reply:
Fixed.